ctapipe_io_lst/containers.py

```diff
@@ -93,16 +93,15 @@ class PedestalContainer(Container):
         'Range of time of the pedestal events [t_min, t_max]',
         unit=u.s
     )
-    n_event = Field(0, 'Number of events used for statistics')
-    pedestal_mean = Field(
+    charge_mean = Field(
         None,
         "np array of pedestal average (n_chan X N_pix)"
     )
-    pedestal_median = Field(
+    charge_median = Field(
         None,
         "np array of the pedestal  median (n_chan X N_pix)"
     )
-    pedestal_rms = Field(
+    charge_rms = Field(
```

Comment:
Is this really the RMS? 

Or is it what physicists regularly call the RMS, but actually is the standard deviation?

So is it calculated as 

sqrt(<x²> - <x>²) or sqrt(<x²>) ?

Reply:
Hi Max,

RMS  (as in high energy physics ;-) ) is intended to be the "root mean square deviation": https://en.wikipedia.org/wiki/Root-mean-square_deviation

for calculating it I am using at present the standard deviation from  numpy

std = sqrt(mean(abs(x - x.mean())**2)

Do you see a problem here?

Reply:
Yes. 

We should not  carry over errors made in PAW in the eighties, that were taken over into ROOT because of "backwards compatibility" into the next generation of frameworks.

RMS = Root Mean Square = sqrt(E(x²)) (https://en.wikipedia.org/wiki/Root_mean_square).

If you calculate the standard devition, please call it standard deviation.

Reply:
See: https://root.cern/root/html534/TH1.html#TH1:GetRMS

>  Note that this function returns the Standard Deviation (Sigma)
  of the distribution (not RMS).
  The name "RMS" was introduced many years ago (Hbook/PAW times).
  We kept the name for continuity.

Reply:
Ok, I can proceed with this change, this will ask also a new PR in
https://github.com/cta-observatory/cta-lstchain/pull/92